src/main/resources/templates/record.html

```diff
@@ -3,56 +3,91 @@
-            Record
-            <th:block th:utext="${content.primaryKey}"></th:block>
-          </h1>
-          <p th:if="${#bools.isTrue(registerId.value() == 'register')}">
-            <a th:href="${registerResolver.baseUriFor(new uk.gov.register.core.RegisterName(content.primaryKey))}" th:utext="${'View ' + content.primaryKey + ' register'}">View this register.</a>
-          </p>
-          <div th:include="fragments/data-formats.html::data-formats"></div>
-          <details role="group">
-            <summary role="button" aria-controls="details-content-0" aria-expanded="false">
-              <span class="summary">What are the table headers?</span>
-            </summary>
-            <div class="panel panel-border-narrow" id="details-content-0" aria-hidden="true">
-              <div th:replace="fragments/field-list.html::field-list(${content.fields})"></div>
+        <div th:replace="fragments/phase-banner.html::phase"></div>
+
+        <div class="breadcrumbs">
+            <ol>
+                <li>
+                    <a href="/" th:utext="${friendlyRegisterName}">Home</a>
+                </li>
+                <li>
+                    <a href="/records">Records</a>
+                </li>
+                <li th:utext="${'Record ' + content.primaryKey}"></li>
+            </ol>
+        </div>
+
+        <div class="grid-row">
+            <div class="column-two-thirds">
+                <h1 class="heading-large">
+                    Record
+                    <th:block th:utext="${content.primaryKey}"></th:block>
+                </h1>
+                <p th:if="${#bools.isTrue(registerId.value() == 'register')}">
+                    <a th:href="${registerResolver.baseUriFor(new uk.gov.register.core.RegisterName(content.primaryKey))}"
+                       th:utext="${'View ' + content.primaryKey + ' register'}">View this register.</a>
+                </p>
+                <div th:include="fragments/data-formats.html::data-formats"></div>
+                <details role="group">
+                    <summary role="button" aria-controls="details-content-0" aria-expanded="false">
+                        <span class="summary">What are the table headers?</span>
+                    </summary>
+                    <div class="panel panel-border-narrow" id="details-content-0" aria-hidden="true">
+                        <div th:replace="fragments/field-list.html::field-list(${content.fields})"></div>
+                    </div>
+                </details>
             </div>
-          </details>
+            <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
         </div>
-        <div class="column-third" th:include="fragments/attribution.html::attribution"></div>
-      </div>
 
-      <div th:include="fragments/item-table.html :: item-table (content = ${content.content})" class="table-wrapper"></div>
+        <table th:if="${!content.content.isEmpty()}"
```

Comment:
The spacing between the table and the surrounding crest, field-list etc. looks a bit dodgy. I think we need a `<div class="table-wrapper">` around it like we do in `records.html`.

Also, it looks like your indentation size must be set to 4 for html - previously it has been size 2. I think 2 is the standard for html so please could you make sure you use 2 so that it is easier to see what has actually changed - currently it looks like you have changed every single line and some html files have different size indentation to others.

It looks like `record.html` and `records.html` have duplicate code for the table. Would it be possible to move these out into a commonly used fragment like has been done with `item-table.html`.